### PR TITLE
get recent submission in editor

### PR DESCRIPTION
**Pull request by talha8045**

### CODE REVIEW

**Comment by flozender** [comment]
Can you also get the Backend code in the PR

File: IDE/js/ide.js [hunk 1]
```diff
@@ -838,6 +838,32 @@ Content of compiled binary is Base64 encoded and used as source code.\n\
...
+    type: "GET",
+    async: true,
+    headers: {
+      authorization: getCookie("token"),
+    },
+    success: function (data) {
+      var a = data.length-1;
+      currentLanguageId = data[a].languageId;
+      sources[currentLanguageId] = data[a].sourceCode;
+      callinsertTemplate(data);
```
Comment (flozender) [inline IDE/js/ide.js:860]:
```suggestion
      insertUserCode(data[a].languageId);
```

File: IDE/js/ide.js [hunk 2]
```diff
@@ -838,6 +838,32 @@ Content of compiled binary is Base64 encoded and used as source code.\n\
 https://ide.judge0.com/?kS_f\n\
 ';
 
+function callinsertTemplate(data){
+  insertTemplate();
```
Comment (flozender) [inline IDE/js/ide.js:842]:
```suggestion
  currentLanguageId = parseInt(data);
  sourceEditor.setValue(sources[currentLanguageId]);
  monaco.editor.setModelLanguage(
    sourceEditor.getModel(),
    $selectLanguage.find(":selected").attr("mode")
  );
  $(".lm_title")[0].innerText = fileNames[currentLanguageId];
```

File: IDE/js/ide.js [hunk 3]
```diff
@@ -838,6 +838,32 @@ Content of compiled binary is Base64 encoded and used as source code.\n\
 https://ide.judge0.com/?kS_f\n\
 ';
 
+function callinsertTemplate(data){
```
Comment (flozender) [inline IDE/js/ide.js:841]:
```suggestion
function insertUserCode(data){
```

**Review by flozender — CHANGES_REQUESTED** [review]

File: IDE/js/ide.js [hunk 4]
```diff
@@ -838,6 +838,38 @@ Content of compiled binary is Base64 encoded and used as source code.\n\
...
+  $.ajax({
+    url: serverUrl + "/submissions/user/" + username.toLowerCase() + "/" + questionId + "/",
+    type: "GET",
+    async: true,
+    headers: {
+      authorization: getCookie("token"),
+    },
+    success: function (data) {
+      var a = data.length-1;
+      currentLanguageId = data[a].languageId;
```
Comment (flozender) [inline IDE/js/ide.js:864]:
```suggestion
      currentLanguageId = parseInt(data[a].languageId);
```

**Comment by flozender** [comment]
Thanks @talha8045!

**Comment by talha8045** [comment]
Thanks for guiding me:)

**Pull request merged**